pdns/dnsdistdist/dnsdist-lua-ffi.cc

```diff
@@ -1119,6 +1119,27 @@ bool dnsdist_ffi_dnsquestion_add_proxy_protocol_values(dnsdist_ffi_dnsquestion_t
   return true;
 }
 
+size_t dnsdist_ffi_dnsquestion_get_proxy_protocol_values(dnsdist_ffi_dnsquestion_t* dnsQuestion, const dnsdist_ffi_proxy_protocol_value_t** out)
+{
+  size_t count = 0;
+  if (dnsQuestion == nullptr || dnsQuestion->dq == nullptr || out == nullptr || !dnsQuestion->dq->proxyProtocolValues) {
```

Comment:
You might want to consider to return the length even if `out` is a `nullptr`, if it makes sense to be able to retrieve the length without actually getting the vector.

Reply:
Makes sense :+1: